### PR TITLE
add integration tests fixtures for post and query string

### PR DESCRIPTION
**Pull request by Pantani**
closes https://github.com/trustwallet/blockatlas/issues/541

Now we can add a body or a query string to do request in the integration tests

- `testdata` - Folder contains the JSON fixtures
- `body_fixtures.json` - Body fixtures for POST requests
- `coin_fixtures.json` - Coin fixtures for path parameters
- `query_fixtures.json` - Query string for GET requests
- `exclude.json` - API's need to be excluded from integration tests

### CODE REVIEW

File: pkg/integration/testdata/query_fixtures.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "/ns/lookup": {
```
Comment (hewigovens) [inline pkg/integration/testdata/query_fixtures.json]: we only test http 200 code?

Comment (vikmeup) [reply pkg/integration/testdata/query_fixtures.json]: would be test to test response result

**Comment by vikmeup** [comment]
In order to complete https://github.com/trustwallet/blockatlas/issues/541, we need to test: .ens, .crypto, .zil domain names, for ethereum and other coins, for both existing and non existing names and make results are correct

**Comment by Pantani** [comment]
> In order to complete #541, we need to test: `.ens`, `.crypto`, `.zil` domain names, for ethereum and other coins, for both existing and non existing names and make results are correct

@vikmeup Done!!!

**Comment by vikmeup** [comment]
@Pantani as I understand those tests only check for 200, it does not tests what it needs to. 

https://github.com/trustwallet/blockatlas/issues/541 need to test different services for their responses data, not just the status code.

**Pull request merged**